src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmployeeController.php

```diff
@@ -478,6 +480,13 @@ protected function getEmployeeFormHandler()
     protected function getErrorMessages(Exception $e)
     {
         return [
+            UploadedImageConstraintException::class => $this->trans(
+                'Image format not recognized, allowed formats are: %s',
+                'Admin.Notifications.Error',
+                [
+                    implode(', ', ImageManagerCore::MIME_TYPE_SUPPORTED),
```

Comment:
I think it might be better to use the class `ImageManager` instead of `ImageManagerCore` so if there's a `ImageManager` override that adds/removes a mime type, this wording could be updated as well
```suggestion
                    implode(', ', ImageManager::MIME_TYPE_SUPPORTED),
```

Reply:
Seems like a good idea @atomiix. I'll take care of it

Reply:
I squash commits because 7 commits just to a an error message sound's a little bit too much :)